hydroshare/settings.py

```diff
@@ -169,11 +169,16 @@ def _(s): return s # noqa
 # is not that great for our project use case
 FILE_UPLOAD_MAX_MEMORY_SIZE = 0
 
+# Wether to enable OIDC auth via mozilla_django_oidc
+# Default false to enable local development
+ENABLE_OIDC_AUTHENTICATION = False
```

Comment:
@sblack-usu I decided to just conditionally add all the oidc settings based on `ENABLE_OIDC_AUTHENTICATION`